src/configure.ac

```diff
@@ -281,6 +281,22 @@ AC_SUBST(CRYPTO_IMPL)
 AC_SUBST(CRYPTO_IMPL_CFLAGS)
 AC_SUBST(CRYPTO_IMPL_LIBS)
 
+if test "$CRYPTO_IMPL" = openssl; then
+    AC_LINK_IFELSE([AC_LANG_SOURCE([
+#include <openssl/kdf.h>
+int main()
+{
+     EVP_KDF *kdf = EVP_KDF_fetch(NULL, "KRB5KDF", NULL);
+     return 0;
+}
```

Comment:
Are we trying to specifically test for the "KRB5KDF" value working?  If so, we need a runtime test (which needs an accompanying default for cross-compilation).

Reply:
I really am just trying to check whether the function exists.  I could check for the values we need at buildtime too if you want.